includes/AMP/Sanitization.php

```diff
@@ -70,6 +70,8 @@ public function sanitize_document( Document $document ) {
 	/**
 	 * Adds missing scripts.
 	 *
+	 * @SuppressWarnings(PHPMD)
```

Comment:
Do need to suppress everything here?

Reply:
Opted for this way here because this code is mostly copied from the AMP plugin.